dataland-frontend/src/components/forms/parts/kpiSelection/DataPointFormField.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <template v-if="evidenceDesired">
+    <FormKit type="group" :name="name">
+      <div class="mb-3">
+        <UploadFormHeader :label="label" :description="description ?? ''" :is-required="required" />
+        <div class="next-to-each-other">
+          <FormKit
+            type="text"
+            name="value"
+            v-model="currentValue"
+            :validation-label="validationLabel ?? label"
+            :validation="`number|${validation}`"
+            placeholder="Value"
+            outer-class="short"
+          />
+          <div v-if="unit" class="form-field-label pb-3">
+            <FormKit type="hidden" name="unit" :modelValue="unit" />
+            <span>in {{ unit }}</span>
+          </div>
+          <FormKit
+            v-else-if="options"
+            type="select"
+            name="unit"
+            v-model="currentUnit"
+            placeholder="Currency"
+            :options="options"
+            outer-class="short"
+          />
+        </div>
+      </div>
+      <div>
+        <FormKit type="group" name="dataSource">
+          <div class="next-to-each-other">
+            <div class="flex-1">
+              <UploadFormHeader :label="`${label} Report`" :description="'Upload Report'" />
```

Comment:
The description doesn't really make sense, since the user cannot upload anything here.
It should say something like "Select a report from the uploaded reports as a reference for this data point." or similar.

Reply:
changed